src/oc/storage/resources/common.clj

```diff
@@ -112,6 +112,7 @@
   (schema/optional-key :video-transcript) (schema/maybe schema/Str)
   (schema/optional-key :video-processed) (schema/maybe schema/Bool)
   (schema/optional-key :video-error) (schema/maybe schema/Bool)
+  (schema/optional-key :revision-id) schema/Int
```

Comment:
@thepug why optional?

Reply:
This was to get around an schema error when the web client made a post edit. I'll look to see how to handle this better.  Probably I only need to pass in `:revision-id` when making edits.

Reply:
The client could send the `:revision-id` and the error would go away but I'm not sure that is ideal. The other solution might be to have an update schema and validation for the rest.  

So the main reason is to stay backwards compatible.

Reply:
@thepug not sure I get why it's a backwards compatibility issue. The migration set 0 as an ID on all existing entries. When will anything need to be backward compatible w/ not having an ID?

Reply:
The client removes the `:revision-id` attribute when posting an edit.  The schema fails because the data doesn't have the `:revision-id`.  If we make the attribute optional, then anything posting to the API can skip this attribute (including the existing web client).  We can release the versions change without changing the client (backwards compatible...but not really).

Reply:
Ok, the last commit ca8a4aa   should address the issue I was seeing.  I didn't realize what the `->entry` function was accomplishing.